cli/flox/doc/flox-activate.md

```diff
@@ -22,8 +22,12 @@ flox [<general-options>] activate
 
 # DESCRIPTION
 
-Sets environment variables and aliases, runs hooks, starts services,
-and adds `bin` directories to your `$PATH`.
+Configures a shell with everything defined by the environment:
+
+* Downloads packages and adds their `bin` directories to your `$PATH`.
+* Sets environment variables and aliases.
+* Runs hooks.
+* Starts services.
```

Comment:
Nit: it doesn't start services by default

Reply:
Good point, though that has been wrong before already. Maybe say "Starts services. (If `--start-services` is given.)"

Reply:
Added. Technically they are all conditional based on whether something is defined in the manifest, but services are the only one that's also opt-in via a flag, so we can call that out.